amundsen_application/static/js/components/common/Flag/index.tsx

```diff
@@ -20,7 +20,7 @@ export function convertText(str: string, caseType: string): string {
     case CaseType.LOWER_CASE:
       return str.toLowerCase();
     case CaseType.SENTENCE_CASE:
-      return `${str.charAt(0).toUpperCase()}${str.slice(1).toLowerCase()}`;
+      return str.split(new RegExp('\\s+')).map(x => x.charAt(0).toUpperCase() + x.slice(1).toLowerCase()).join(" ");
```

Comment:
Should I create a new CaseType for PascalCase? Or is it ok modifying SENTENCE_CASE to produce:

"test example" -> "Test Example"

Reply:
I hadn't intended for this component to be modified. Rather this was an example of what `EditableSection` could do to create the `title` to ensure consistent titles in `Title Case`. The concept here is that:
1. We have `source` passed in from the backend, and an optional configuration for the display name
2. The UI requirement is that the `EditableSection` titles be rendered in `Title Case`
3. I want to decrease the need for configuring the display name, and one way to do this is to establish/expect a consistent pattern for `source` such that `EditableSection` can take values like "quality_service" and "s3_crawler", then automatically render "Quality Service" and "S3 Crawler" as the title without the engineer needing to configure anything.

Reply:
Ok so I think I have addressed the main concerns now:
- no more configurable display title
- source is used and made to conform to requirements

The only thing I am unsure if you are ok is if I can reuse that convert text function with a new case. Or would you prefer I just add a new function in the EditableSection index?

Reply:
How about add a new function and we can clean it up later